api/src/main/java/org/ehcache/spi/service/PersistableResourceService.java

```diff
@@ -49,6 +50,15 @@
   Collection<ServiceConfiguration<?>> additionalConfigurationsForPool(String alias, ResourcePool pool) throws CachePersistenceException;
 
   /**
+   * Create the persistence space with the given name.
+   *
+   * @param name the name of the persistence context
+   * @param config the configuration for the associated cache
+   * @throws CachePersistenceException if the persistence space cannot be created
+   */
+  void create(String name, CacheConfiguration<?, ?> config) throws CachePersistenceException;
```

Comment:
The `create` method is going to be removed from the `PersistableResourceService` interface -- seems it can be local to `DefaultLocalPersistenceService` and _may_ actually be removable entirely.